piksi_tools/console/baseline_view.py

```diff
@@ -180,6 +180,7 @@ def baseline_callback(self, sbp_msg):
 
       if self.log_file is None:
         self.log_file = open(time.strftime("baseline_log_%Y%m%d-%H%M%S.csv"), 'w')
+        self.log_file.write('time,meters_north,meters_east,meters_down,distance(meters),num_sats,flags,num_hypothesis')
```

Comment:
maybe change meters_north -> north(meters) to be consistent with the other fields (and same with east, down)?